custom_components/tuya_local/devices/woox_r4028_powerstrip.yaml

```diff
@@ -0,0 +1,83 @@
+name: Woox R4028 + DIGOO DG-PS01
+primary_entity:
+  entity: switch
+  name: switch 1
```

Comment:
To be consistent with the other supported multi-switch device and avoid adding new translations unnecessarily, the name should be `Outlet 1` (similarly for 2 and 3)